extensions/html/src/html.rs

```diff
@@ -2,7 +2,7 @@ use std::{env, fs};
 use zed_extension_api::{self as zed, Result};
 
 const SERVER_PATH: &str =
-    "node_modules/vscode-langservers-extracted/bin/vscode-html-language-server";
+    "/Users/hiro/Projects/vscode-langservers-extracted/bin/vscode-html-language-server";
```

Comment:
Hardcoded path here still? (Sorry if you were already aware of it.)

Reply:
No worries; yeah, this PR is still WIP